src/routes/index.js

```diff
@@ -1,15 +1,18 @@
 import { Router } from 'express';
 import swaggerUI from 'swagger-ui-express';
-import welcomeRoute from './welcome';
-import Badroute from './badroute';
+// import welcomeRoute from './welcome';
+// import Badroute from './badroute';
```

Comment:
@ishimwebernard you shouldn't other people's features. This is a big no no... you should make your code increment work well with the already existing work.